alibi_detect/utils/statstest.py

```diff
@@ -1,17 +1,9 @@
-import dask.array as da
 import numpy as np
 from typing import Callable, Tuple, Union
-from alibi_detect.utils.distance import maximum_mean_discrepancy
 
 
-def permutation_test(x: Union[np.ndarray, da.array],
-                     y: Union[np.ndarray, da.array],
-                     n_permutations: int = 1000,
-                     metric: Callable = maximum_mean_discrepancy,
-                     return_distance: bool = False,
-                     return_permutation_distance: bool = False,
-                     **kwargs) \
-        -> Union[np.float, Tuple[np.float, np.float], Tuple[np.float, np.float, np.ndarray]]:
+def permutation_test(x: np.ndarray, y: np.ndarray, metric: Callable, n_permutations: int = 100,
```

Comment:
Why remove `dask` type hints? More generally, if it's true that all of our `numpy` functions would work seamlessly with `dask` arrays, we could not bother with `dask` typehints, especially if users never explicitly deal with `dask` arrays.